src/main/kotlin/no/nav/familie/ba/sak/kjerne/dokument/DokumentService.kt

```diff
@@ -155,27 +160,35 @@ class DokumentService(
                                                                       behandlingId = behandling.id)
         }
 
-        return distribuerBrevOgLoggHendelse(journalpostId = journalpostId,
-                                            behandlingId = behandling.id,
-                                            loggTekst = manueltBrevRequest.brevmal.visningsTekst.replaceFirstChar { it.uppercase() },
-                                            loggBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
-                                            brevType = manueltBrevRequest.brevmal.tilSanityBrevtype())
+        DistribuerDokumentTask.opprettDistribuerDokumentTask(
+                distribuerDokumentDTO = DistribuerDokumentDTO(
+                        personIdent = behandling.fagsak.hentAktivIdent().ident,
+                        behandlingId = behandling.id,
+                        journalpostId = journalpostId,
+                        brevType = manueltBrevRequest.brevmal.tilSanityBrevtype(),
+                ),
+                properties = Properties().apply {
+                    this["fagsakIdent"] = behandling.fagsak.hentAktivIdent().ident
+                    this["mottakerIdent"] = manueltBrevRequest.mottakerIdent
+                    this["journalpostId"] = journalpostId
+                    this["behandlingId"] = behandling.id.toString()
+                }
+        ).also {
```

Comment:
```suggestion
        )copy(triggerTid = nesteGyldigeTriggertidForBehandlingIHverdager()).also {
```

Dette er kanskje ikke nødvendig 🤷‍♂️